### ISSUE TITLE
Rubocop Target Ruby Version

### ISSUE THREAD

**Issue by olbrich** [ajgon/opsworks_ruby#273]
https://github.com/ajgon/opsworks_ruby/blob/e9b2ac5b222f0764d027a66435f7349a3ae66d5d/.rubocop.yml#L358

So Rubocop targets ruby 2.6, and is used to lint code in the cookbooks. From what I can tell, the cookbooks are actually executed using the embedded ruby version with the Chef client, which is more like ruby 2.3.1.  Targeting a newer version could lead to usage of language features and syntax that won't actually work on cookbooks when they are deployed.  The same is true re: the `.ruby-version` file, which currently lists ruby 2.7.1.  

BTW, rubocop will fall back to using the .ruby-version file to determine the targeted ruby version if one is not specified.

**Comment by ajgon** [comment]
I agree, rubocop should check against `2.3.1` - I'll fix that. Fallback is not an issue, as we use it directly in `.rubocop.yml`, but I wasn't aware of that - good to know :+1:
